package.json

```
@@ -6,7 +6,9 @@
   "scripts": {
     "commitlint": "commitlint --from d470006",
     "test": "nyc mocha -- 'tests/**/*.test.js'",
-    "standards": "npm run commitlint && grunt standards"
+    "markdownlint": "markdownlint -c .markdownlint.json -i CHANGELOG.md '{,!(node_modules)/**/}*.md'",
+    "eslint": "eslint '{,!(node_modules|dist)/**/}*.js'",
+    "standards": "npm run markdownlint && npm run eslint"
```

Seems this might be missing `npm run commitlint`

```suggestion
    "standards": "npm run commitlint && npm run eslint && npm run markdownlint"
```